mantidproject/mantid
# Add folder to the docs-qthelp target (only affects VS)

**Description of work.**

Move the docs-qthelp target back into the Documentation folder (only affects Visual Studio)

Also fix problem with import readline that was preventing docs build on Windows. I think this module should only be imported on Linux. Have wrapped it in an "if sys.platform" statement that I saw used in qt\applications\workbench\workbench\app\start.py

**To test:**

Run cmake on Windows and check the target shows up in the Documentation folder in Visual Studio:

![image](https://user-images.githubusercontent.com/56295817/99107765-3c536d80-25de-11eb-8eb9-266b4c65b901.png)

Build the docs-qtassistant target and then the docs-qthelp target and verify that the docs have built OK and look OK when you view the docs in Workbench

Fixes #29957.

*This does not require release notes* because **fixes a non-user problem that only appeared in this release**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
